src/gleam/bit_array.gleam

```diff
@@ -193,3 +193,48 @@ fn do_inspect(input: BitArray, accumulator: String) -> String {
     _ -> accumulator
   }
 }
+
+/// Compare two bit arrays as sequences of bytes.
+///
+/// ## Examples
+///
+/// ```gleam
+/// compare(<<1>>, <<2>>)
+/// // -> Lt
+///
+/// compare(<<"AB":utf8>>, <<"AA":utf8>>)
+/// // -> Gt
+///
+/// compare(<<1, 2:size(2)>>, with: <<1, 2:size(2)>>)
+/// // -> Eq
+/// ```
+///
+/// Only supported on Erlang target for now.
+///
+pub fn compare(a: BitArray, with b: BitArray) -> order.Order {
+  do_compare(a, b)
```

Comment:
The do_compare helper doesn't do anything here, the implementation can be in this function